index.html

```diff
@@ -418,113 +418,112 @@ <h3 class="h3 card-title">24/7 hours support</h3>
         - #PROJECT
       -->
 
-        <section class="section project" id="project" aria-label="project">
-          <div class="container">
+      <section class="section project" id="project" aria-label="project">
+        <div class="container">
 
-            <p class="section-subtitle has-before text-center">Projects</p>
+          <p class="section-subtitle has-before text-center">Projects</p>
 
-            <h2 class="h2 section-title text-center">
-              MusiTech complete <span class="has-before">project</span>
-            </h2>
+          <h2 class="h2 section-title text-center">
+            MusiTech complete <span class="has-before">project</span>
+          </h2>
 
-            <ul class="grid-list">
+          <ul class="grid-list">
 
-              <li>
-                <div class="project-card">
+            <li>
+              <div class="project-card">
 
-                  <figure class="card-banner img-holder" style="--width: 835; --height: 429;">
-                    <img src="./assets/images/project-1.jpg" width="835" height="429" loading="lazy"
-                      alt="Book art design" class="img-cover">
-                  </figure>
+                <figure class="card-banner img-holder" style="--width: 835; --height: 429;">
+                  <img src="./assets/images/webdev.avif" width="835" height="429" loading="lazy"
+                    alt="Book art design" class="img-cover">
+                </figure>
 
-                  <div class="card-content">
-                    <h3 class="h3">
-                      <a href="#" class="card-title">Niche Studio Website</a>
-                    </h3>
-                    <!-- <a href="#" class="card-tag">Branding</a> -->
-                    <!-- HTML !-->
-                    <button class="button-48" role="button"><span class="text">Check Out</span></button>
-                  </div>
+                <div class="card-content">
+                  <h3 class="h3">
+                    <a href="#" class="card-title">Web Development</a>
+                  </h3>
+                  <!-- <a href="#" class="card-tag">Branding</a> -->
+                  <!-- HTML !-->
+                  <button class="button-48" role="button" onclick="openInNewTab('https://notevid.vercel.app/')"><span class="text">Check Out</span></button>
```

Comment:
Instead of writing a js function, can't we wrap the `button` in `<a href = " "></a>` directly?

Reply:
yes.